umap/tests/integration/test_statics.py

```diff
@@ -24,7 +24,7 @@ def test_javascript_have_been_loaded(
 ):
     settings.STORAGES["staticfiles"][
         "BACKEND"
-    ] = "umap.storage.UmapManifestStaticFilesStorage"
+    ] = "umap.storage.UmapManifestStaticFilesStorageNotStrict"
```

Comment:
I don't think this line is even used: given the staticfiles is run in a fixture, changing the setting at this time will not do anything. So I think the current test is already broken.